### DISCUSSION TITLE
如何获取图片资源，不下载到本地

### DISCUSSION THREAD

**Discussion by paiap** [didi/LogicFlow#1184] [Q&A] [answered]
我想整个实例保存后，将数据提交给后端，同时我还需要将当前结构图也上传至后端，（获取图片资源，然后上传），请问如何调用可以获取图片资源？使用getSnapshot方法是直接下载到本地了

**Comment by towersxu** [comment, marked as answer]
可以使用getGraphData api获取json格式的数据。具体流程可以参考项目https://github.com/logic-flow/turbo-client
